common/proto-options/solana/get_public_keys.options

```diff
@@ -0,0 +1,5 @@
+# Options for file common/cypherock-common/proto/solana/get_public_key.proto
+solana.GetPublicKeysDerivationPath.path type:FT_STATIC max_count:5 fixed_length:true
```

Comment:
Update max count to 4 as max depth for solana is 4

Reply:
resolving as this is updated.